### PR TITLE
Changed way of reading tags without values

### PR DESCRIPTION
**Pull request by bartekzylinski**
closing #511

### CODE REVIEW

File: rest/rest-sqs/src/test/scala/org/elasticmq/rest/sqs/TagsTest.scala [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package org.elasticmq.rest.sqs
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class TagsModuleTest extends AnyFlatSpec with AttributesModule with Matchers {
+
+  "Empty value for tag" should "be read as empty string" in {
```
Comment (adamw) [inline rest/rest-sqs/src/test/scala/org/elasticmq/rest/sqs/TagsTest.scala:8]: can we add some tests for non-empty values as well?

File: rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/TagsModule.scala [hunk 2]
```diff
@@ -46,3 +49,5 @@ trait TagsModule {
     }
   }
 }
+
+object TagsModule extends TagsModule
```
Comment (adamw) [inline rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/TagsModule.scala:53]: let's just use `new TagsModule {}` in tests, since this object isn't used anywhere in production code

**Comment by adamw** [comment]
👍

**Comment by adamw** [comment]
Thanks!
